metrics/build.gradle

```diff
@@ -0,0 +1,37 @@
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion project.ext.targetSdkVersion
+
+    defaultConfig {
+        minSdkVersion project.ext.minSdkVersion
+        targetSdkVersion project.ext.targetSdkVersion
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+
+        // Required for metrics, exposed by the parent project
+        versionName project.ext.versionName
```

Comment:
I think it should be the core responsibility

Reply:
Using Core for versioning on IOS